dvc/scm/git.py

```diff
@@ -180,7 +180,8 @@ def untracked_files(self):
         return [os.path.join(self.git.working_dir, fname) for fname in files]
 
     def is_tracked(self, path):
-        return len(self.git.git.ls_files(path)) != 0
+        path = os.path.relpath(path, self.root_dir)
+        return path in [i[0] for i in self.git.index.entries]
```

Comment:
out of curiosity - what is the difference between these two implementations?

Reply:
`ls_files` fails with unicode filenames